tests/migration_test.go

```diff
@@ -1639,8 +1639,8 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				vmi = tests.NewRandomVMIWithPVC(pvName)
 				vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse(fedoraVMSize)
 				vmi.Spec.Domain.Devices.Rng = &v1.Rng{}
-
 				tests.AddUserData(vmi, "cloud-init", "#!/bin/bash\n echo hello\n")
+
```

Comment:
Can you please drop these newline changes? It is confusing.

Reply:
Why do you think it's confusing? I just divided up the parts where I set up the VMI from the part when I run it, it actually reads better for me.

Reply:
As a reviewer I am looking for changes related to the intentions described in the commit message. If I see changes, it takes me time to understand that they do nothing. I prefer to see only the things that relate to what is relevant to ease the review. I usually do not mind if it is close to some other code changed, but this one is not.

I also think that it is a more organized to do it in a separate commit, helping future git history readers see only the relevant changes.